### ISSUE TITLE
Update or remove missingkey documentation

### ISSUE THREAD

**Issue by joebowbeer** [helm/helm#11229]
The missingkey option [documented in tips and tricks](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-required-function) is settable somehow?

I can't find any examples and AFAICT no one else can figure out how to use it in helm v3 either:

https://github.com/helm/helm/issues/5744#issuecomment-494153879
https://github.com/helm/helm/issues/5744#issuecomment-703652296
https://github.com/helm/helm/issues/5417#issuecomment-473372043
https://github.com/helm/helm/issues/5180#issue-400200752

Related:

https://github.com/helm/helm/issues/9273
https://github.com/helm/helm/issues/7483#issuecomment-620296071

Please update the doc or remove the missingkey reference from tips and tricks.

Output of `helm version`: v3.6.3

**Comment by joejulian** [comment]
I agree that's hard to read. The first paragraph is establishing a behavior that the author expects the reader to be familiar with (go-templates). Then goes on to say, "There may be situations where a chart developer wants to enforce this behavior for select values ..." implying that the following `required` function may be used in templates to enforce the setting of specific keys.

As @bacongobbler has also mentioned, you can also define a json schema to be explicit about your values expectations without resorting to flagging keys all throughout your templates.

PRs to helm-www would be gladly accepted to make this more readable.

@helm/helm-core-maintainers can someone please move this to helm-www
